study-builder/tenants/circadia/tenant.yaml

```diff
@@ -0,0 +1,50 @@
+tenant:
+  default_directory: Username-Password-Authentication
+
+rules:
+  - name: Register User in Pepper
+    script: ../register-user-in-pepper.js
+    stage: login_success
+    enabled: true
+    order: 1
+
+pages:
+  - name: login
+    enabled: true
+    html: ./pages/login.html
+  - name: password_reset
+    enabled: true
+    html: ./pages/password_reset.html
+
+emailTemplates:
+  - template: welcome_email
+    enabled: true
+    syntax: liquid
+    from: '##FROM_EMAIL##'
```

Comment:
I don't see `FROM_EMAIL`... if we add `STUDY_EMAIL` maybe we can use that here? Also, I think the format for this `from` value looks something like
```
Study Name <email_here>
```